src/main/resources/config/modules/Vanilla.xml

```diff
@@ -726,9 +726,9 @@
             <BiomeType name='Swamp'  />
             <BiomeType name='Beach'  />
             <BiomeType name='River'  />
-            <Biome name='Ocean'  weight='-1' />
-            <Biome name='Desert'  weight='-1' />
-            <Biome name='Mountain'  weight='-1' />
+            <Biome name='.*Ocean.*'  weight='-1' />
```

Comment:
Draco's right.  The distribution is not restricted to swamp, beach, or river; the configuration is designed to _prefer_ those biomes, which means that, in those biomes, the distribution is applied multiple times to make it plentiful.